app/assets/javascripts/file_manager.js

```diff
@@ -1,41 +1,9 @@
-// Add new row of input
-function injectFileInput() {
-  var new_file_field = jQuery('<input>', {
-    type: 'file',
-    name: 'new_files[]',
-    multiple: true
-  });
-
-  new_file_field.change(function() {
-    sanitized_filename_check(this);
-    return false;
-  });
-
-
-  var new_file_field_row = jQuery('<tr>');
-
-  var new_file_field_input_column = jQuery('<td>', {colspan: 4});
-
-  var remove_new_file_input = jQuery('<input>', {type: 'checkbox'});
+jQuery(document).ready(function (){
+	window.modal_addnew = new ModalMarkus('#addnew_dialog');	
```

Comment:
Identifier 'modal_addnew' is not in camel case.<br>'ModalMarkus' is not defined.